src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -124,6 +117,37 @@ private DatadogHttpClient(String url, String logIntakeUrl, Secret apiKey) {
         this.logIntakeUrl = logIntakeUrl;
     }
 
+    public static void validateCongiguration(String url, String logIntakeUrl, Secret apiKey) throws RuntimeException, IOException {
```

Comment:
Let's not use unchecked exception here. Also we should be able to throw a single exception type

Reply:
Removed!